crm_lead_vat/models.py

```diff
@@ -22,9 +22,10 @@ def _lead_create_contact(self, lead, name, is_company, parent_id=False):
     def on_change_partner_id(self, partner_id):
```

Comment:
That was the first option, but as it is an old onchange it does not work as expected. So this was the real fix.